oauth2-authorization-server/src/main/resources/org/springframework/security/oauth2/server/authorization/client/oauth2_registered_client_h2.ddl

```diff
@@ -0,0 +1,16 @@
+create table oauth2_registered_client (
+    id varchar(128) primary key not null,
+    client_id varchar(128) not null,
+    client_id_issued_at datetime,
```

Comment:
Unfortunately HSQLDB does not support it properly, it ignores default value. So in addition to default value in database, there is additional check in parameters mapper that sets `Instant.now()` timestamp if it is null. It works but it does not look good to me, yet I don't see solution without further complicating things. Let me know if you think it should be further improved.

Reply:
If it's an issue of hsqldb ignoring it, I think that's fine. We still want to include it so we are supporting full database systems that use ANSI standard SQL. Adding code as a fallback is fine, so that it also works with hsqldb. (But hsqldb isn't our target database.)